sktime/series_as_features/tests/test_all_panel_estimators.py

```diff
@@ -66,13 +66,10 @@ def test_3d_numpy_input(Estimator):
 
 @pytest.mark.parametrize("Estimator", CLASSIFIERS + REGRESSORS)
 def test_multivariate_input(Estimator):
+    """Test classifiers handle multivariate pd.DataFrame input correctly."""
     # check if multivariate input is correctly handled
     n_columns = 2
-    error_msg = (
-        f"X must be univariate "
-        f"with X.shape[1] == 1, but found: "
-        f"X.shape[1] == {n_columns}."
-    )
+    error_msg = "X must be univariate"
```

Comment:
why are we changing the error message?

Reply:
hmmm, not sure! I'll put it back and see if anything breaks